test/parallel/test-net-connect-options-fd.js

```diff
@@ -32,7 +33,8 @@ const forAllClients = (cb) => common.mustCall(cb, CLIENT_VARIANTS);
 
 // Test Pipe fd is wrapped correctly
 {
-  const prefix = `${common.PIPE}-net-connect-options-fd`;
+  const prefix = path.relative(`${common.PIPE}-net-connect-options-fd`,
+                               `${__dirname}-net-connect-options-fd`);
```

Comment:
I think this should be this instead:

```js
const prefix = path.relative('.', `${common.PIPE}-net-connect-options-fd`);
```

Simpler, and I *think* it will fix the problems we're seeing on CI.